app/models/katello/concerns/content_facet_host_extensions.rb

```diff
@@ -8,7 +8,9 @@ module ContentFacetHostExtensions
           :security_needed => Katello::ErrataStatus::NEEDED_SECURITY_ERRATA,
           :errata_needed => Katello::ErrataStatus::NEEDED_ERRATA,
           :updated => Katello::ErrataStatus::UP_TO_DATE,
-          :unknown => Katello::ErrataStatus::UNKNOWN
```

Comment:
This is basically an API change. I could keep 'unknown' but map it to ::UNREGISTERED?